docs/index.md

```diff
@@ -51,7 +51,14 @@
 
 ## Acknowledgements
 
-<img src="./images/google_org.png" alt="google_dot_org" width=200/>
-
+<img src="./images/google_org.png" alt="google_dot_org" width=200/><br>
 The project is funded by Google.org through the
 <a href="https://globalgoals.withgoogle.com/globalgoals/" class="link-home">AI for Global Goals</a> grant.
+<br><br>
+<img src="./images/idinsight_org.png" alt="idi_org" width=200 style="margin-left:-25px" /><br>
+Built and powered by IDinsight.
+<p class="footer" markdown>IDinsight uses data and evidence to help leaders combat poverty worldwide. Our collaborations deploy a large analytical toolkit to help clients design better policies, rigorously test what works, and use evidence to implement effectively at scale. We place special emphasis on using the right tool for the right question, and tailor our rigorous methods to the real-world constraints of decision-makers. IDinsight works with governments, foundations, NGOs, multilaterals and businesses across Africa and Asia. We work in all major sectors including health, education, agriculture, governance, digital ID, financial access, and sanitation. We have offices in Dakar, Lusaka, Manila, Nairobi, New Delhi, Rabat, and Remote.
+<br>
+<br>
+:globe_with_meridians: <a href="https://www.idinsight.org" class="link-home">www.idinsight.org</a>
```

Comment:
This is also maybe out of scope for this PR and don't need to address this point, but this light-dark mode colors seem tricky. The yellow looks great in dark mode but a bit low contrast for light mode 🤔 
<img width="404" alt="image" src="https://github.com/IDinsight/aaq-core/assets/7042047/3f6e33e0-db96-4350-b18a-10b2904f6e83">